config/m4/sysdep.m4

```diff
@@ -143,8 +143,13 @@ AC_CHECK_HEADERS([sys/capability.h],
                                  [[#include <sys/capability.h>]])]
                  )
 
-
 #
 # Check for PR_SET_PTRACER
 #
 AC_CHECK_DECLS([PR_SET_PTRACER], [], [], [#include <sys/prctl.h>])
+
+#
+# epoll vs. kqueue
```

Comment:
minor: comment should be "check for epoll support"
i guess kqueue check will be another section